packages/@tests/smoke/web/public/index.js

```diff
@@ -85,7 +85,9 @@ const getRelayTime = () => {
 
 const main = async () => {
   console.log("starting fluence...");
-  fluence.defaultClient = await fluence.clientFactory(relay, {});
+  fluence.defaultClient = await fluence.clientFactory(relay, {
+    CDNUrl: "http://localhost:3000",
```

Comment:
Will URL be reported in an error, if it is not available?

Reply:
It will give you runtime error

Reply:
JS-client will throw when you try to run it

Reply:
will the error message include URL? I think it should